assets/app/view/form.rb

```diff
@@ -69,9 +73,15 @@ def render_input(label, placeholder: '', id:, el: 'input', type: 'text', attrs:
       )
     end
 
-    def render_button(text, &block)
+    def render_button(text, style: {}, &block)
       props = {
-        attrs: { type: :button },
+        attrs: {
+          id: text.gsub(/\s/, '-').downcase,
```

Comment:
i feel like these are just unnecessary, but if you really want them you can keep it, it's just extra code and line 60

Reply:
As I said before: on input an id is needed to connect label and input => clicking on label to activate input won’t work otherwise (if you don’t change the html structure)
On buttons they come in handy if you’d like to address one specific button. Kill L79 if you must.

Reply:
where do you reference the id?

Reply:
labels: for: 
buttons: CSS e. g. (notice the *if* there)

Reply:
can you pass the id in explicitly and not do multiple gsubs?

Reply:
render_input: one gsub would be enough. It shouldn’t be needed under normal circumstances (who puts \s in their ids?), but it’s there to safeguard a viable id.

render_button: Of course, but that would be an additional parameter when calling. This way it gets created without the need for that. It’s just a useful default without much hassle. As I said it may come in handy (or it may not).

Reply:
Just let me know if you want them removed, everything else is done.

Reply:
ok, let's just remove it then if it was just done as a safe guard

Reply:
still waiting on the min_max native changes